docs/examples/cmark_spec_0.30.yml

```diff
@@ -10489,7 +11191,8 @@ cases:
         - type: paragraph
           children:
             - type: link
-              url: "/url\u00A0\"title\""
```

Comment:
A few picky things in here like non-space whitespace. I think the problem here is that the non-space whitespace isn't actually present in the markup...